src/main/java/roomescape/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,21 @@
+package roomescape;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.util.NoSuchElementException;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+
+    @ExceptionHandler(NoSuchElementException.class)
```

Comment:
`@ExceptionHandler`가 붙은 메서드의 파라미터에 해당 예외 타입의 변수가 있을경우,
어노테이션의 파라미터를 생략할 수 있습니다!
```suggestion
    @ExceptionHandler
```